hexa_wexa:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers --------------------------------------
  leg1_position_controller:
    type: effort_controllers/JointPositionController
    joint: leg1
    pid: {p: 10.0, i: 0, d: 0.1}
  leg2_position_controller:
    type: effort_controllers/JointPositionController
    joint: leg2
    pid: {p: 10.0, i: 0, d: 0.1}
  leg3_position_controller:
    type: effort_controllers/JointPositionController
    joint: leg3
    pid: {p: 10.0, i: 0, d: 0.1}
  leg4_position_controller:
    type: effort_controllers/JointPositionController
    joint: leg4
    pid: {p: 10.0, i: 0, d: 0.1}
  leg5_position_controller:
    type: effort_controllers/JointPositionController
    joint: leg5
    pid: {p: 10.0, i: 0, d: 0.1}
  leg6_position_controller:
    type: effort_controllers/JointPositionController
    joint: leg6
    pid: {p: 10.0, i: 0, d: 0.1}
  femur1_position_controller:
    type: effort_controllers/JointPositionController
    joint: femur1
    pid: {p: 10.0, i: 0, d: 0.1}
  femur2_position_controller:
    joint: femur2
    pid: {p: 10.0, i: 0, d: 0.1}
    type: effort_controllers/JointPositionController
  femur3_position_controller:
    type: effort_controllers/JointPositionController
    joint: femur3
    pid: {p: 10.0, i: 0, d: 0.1}
  femur4_position_controller:
    type: effort_controllers/JointPositionController
    joint: femur4
    pid: {p: 10.0, i: 0, d: 0.1}
  femur5_position_controller:
    type: effort_controllers/JointPositionController
    joint: femur5
    pid: {p: 10.0, i: 0, d: 0.1}
  femur6_position_controller:
    type: effort_controllers/JointPositionController
    joint: femur6
    pid: {p: 10.0, i: 0, d: 0.1}
